spleeter_service/Dockerfile

```
@@ -0,0 +1,13 @@
+
+FROM python:3.9-slim
+
+WORKDIR /app
+
+# Install Spleeter and required dependencies
+RUN apt-get update && apt-get install -y ffmpeg && apt-get clean
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)